frontend/src/components/Help/HelpNavItem.tsx

```diff
@@ -18,8 +18,9 @@ const HelpNavItem: FunctionComponent = () => {
   return (
     <Dropdown as={Nav.Item} aria-labelledby="navbarDropdownHelp">
       <Dropdown.Toggle as={Nav.Link} id="help-nav-dropdown" data-cy="help">
-        <i className="material-icons">help</i>
-        <p className="d-lg-none d-md-block">Help</p>
+        <Button variant="btn-secondary" className="btn-secondary" size="sm">
```

Comment:
Incorrect variant ... I'm pretty sure this should result in an error.
```suggestion
        <Button variant="secondary" size="sm">
```